### ISSUE TITLE
Biometrics on app launch.

### ISSUE THREAD

**Issue by ascisco** [BlueWallet/BlueWallet#3066]
It seems this feature was removed and biometrics is only enabled for sending funds.
I feel having biometrics for app launch is important for security reasons if someone launches the app and can see how much funds I have. Please re-enable this feature.

Thank you for all your hard work!

**Comment by marcosrdz** [comment]
Hello! Have you enabled storage encryption ? This is the only scenario where biometrics is not used for unlocking the wallets since the the password is not stored on device

**Comment by ascisco** [comment]
Storage encryption is disabled. Before I updated the app it used to ask for finger scan at startup, now it only asks when sending funds.

**Comment by marcosrdz** [comment]
When you get the chance, go into Settings, Security. Disable biometrics and then reenable it. Kill the app and relaunch it. Let us know if that fixes the issue.

**Comment by ascisco** [comment]
Tried it but the problem still persists.

**Comment by marcosrdz** [comment]
Would you mind sharing your device make, model and OS version?

**Comment by ascisco** [comment]
Pixel 4a w/ latest Android OS

**Comment by marcosrdz** [comment]
@ncoelho  can you try to replicate this issue?

**Comment by ascisco** [comment]
This issue still persists.

**Comment by marcosrdz** [comment]
Any chance you can provide a screen recording of the whole process from start to finish? There could be a visual clue that We can use to debug

**Comment by Overtorment** [comment]
is it still happening?

**Comment by ascisco** [comment]
> is it still happening?

Yes. I uninstalled the app and reinstalled it, the biometrics worked on the first launch after setting it up, but afterwards it never asked again. 
Just to reiterate, I am prompted with biometric confirmation when I am confirming to send funds, but it does not prompt me when opening the app.

**Comment by marcosrdz** [comment]
Ive been trying to replicate with  multiple devices with no luck. Any chance you can screen record a video ? From fresh start, to going into the security setting screen and enabling/disabling it. then killing the app and relaunching it?
This could help us see any visual hints to solve the issue

**Comment by ascisco** [comment]
I tried to do a screen recording but none of the apps I tried record the screen properly. It doesnt mirror the screen properly, the fingerprint prompt when enabling biometric doesn't show up in the playback of the screen recording.

After doing a few uninstalls and reinstalls, bluewallet prompted me twice at app open for fingerprint but then stopped prompting me afterwards. Other times it doesnt prompt me at all and other times it only prompts me once.

I also just tried installing bluewalllet and checking on my old pixel 1 and the same problem is persists. 

I am downloading the lastest APK from this github page.

**Comment by mcsaeid** [comment]
Same issue by a user on Twitter ([Tweet](https://twitter.com/ReihaniHamed/status/1426669882565869568))

He did as @marcosrdz asked: Disable and re-enable biometrics, then quit and relaunch the app—no luck.

BlueWallet 6.2.2
Phone: Xiaomi Mi 9T Pro
OS: Android 11 (MIUI 12.5)

**Comment by marcosrdz** [comment]
Issue resolved?
